modules/BlueskyClip/ViewController.swift

```diff
@@ -66,23 +66,51 @@ class ViewController: UIViewController, WKScriptMessageHandler, WKNavigationDele
     guard let url = navigationAction.request.url else {
       return .allow
     }
-
+    
     // Store the previous one to compare later, but only set starterPackUrl when we find the right one
     prevUrl = url
     // pathComponents starts with "/" as the first component, then each path name. so...
     // ["/", "start", "name", "rkey"]
-    if url.pathComponents.count == 4,
-       url.pathComponents[1] == "start" {
+    if isStarterPackUrl(url){
       self.starterPackUrl = url
     }
-
+    
     return .allow
   }
+  
+  func isStarterPackUrl(_ url: URL) -> Bool {
+    var host: String?
+    if #available(iOS 16.0, *) {
+      host = url.host() ?? ""
+    } else {
+      host = url.host
+    }
+    
+    switch host {
+    case "bsky.app":
+      if url.pathComponents.count == 4,
+         (url.pathComponents[1] == "start" || url.pathComponents[1] == "starter-pack") {
+        return true
+      }
+      return false
+    case "go.bsky.app":
+      if url.pathComponents.count == 2 {
+        return true
+      }
+      return false
+    default:
+      return false
+    }
+  }
```

Comment:
This will catch all possibilities: go.bsky.app, bsky.app/start, and bsky.app/starter-pack.